AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java

```diff
@@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the catalog object which will start and play the video.
         Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
```

Comment:
Not what I am looking for.  I want the ability to say "give me a catalog
ugly id for the video named 'Return of Sintel', my video cloud id is
"jfafja;lja;rjkl;rejke;lj" or some such.

On Sat, May 30, 2015 at 10:48 PM Billy Hnath <notifications@github.com>
wrote:

> In
> AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java
> <https://github.com/BrightcoveOS/android-player-samples/pull/85#discussion_r31386306>
> :
>
> > @@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
> >
> >          // Create the catalog object which will start and play the video.
> >          Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
>
> strings.xml:
>
>
> name="catalog_api_read_token">"FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA.."
>
> MainActivity.java:
>
> Catalog catalog = new Catalog(R.string.catalog_api_read_token);
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BrightcoveOS/android-player-samples/pull/85/files#r31386306>
> .
>